rasa/shared/utils/schemas/domain.yml

```diff
@@ -47,6 +47,8 @@ mapping:
       regex;([A-Za-z]+):
         type: "map"
         mapping:
+          regex;([A-Za-z]+):
```

Comment:
Are we sure that this doesn't short-circuit the other lower one? i.e. do we still enforce that `influence_conversation` is type `bool`?

Reply:
I'll check & also add a test that the validator yells when type is wrong

Reply:
It is still enforced, added tests 👍